com.zsmartsystems.zigbee.console/src/main/java/com/zsmartsystems/zigbee/console/ZigBeeConsoleAbstractCommand.java

```diff
@@ -217,6 +262,16 @@ protected String printZclDataType(ZclDataType dataType) {
         return String.format("%-25s", dataType);
     }
 
+    /**
+     * @param cluster a ZCL cluster
+     * @return a String containing information about the cluster, example 'OnOff server cluster 0x00A4'
+     */
+    protected String printCluster(ZclCluster cluster) {
+        String typePrefix = cluster.isServer() ? "server " : "client ";
```

Comment:
Are the spaces needed here? It will add an extra space in the output and therefore won't be what you list above.